src/firebaseConfig.ts

```diff
@@ -102,13 +102,14 @@ export type FirestoreConfig = {
   indexes?: string;
 } & Deployable;
 
-export type FunctionsConfig = {
-  // TODO: Add types for "backend"
+export type FunctionsSingle = {
   source?: string;
   ignore?: string[];
   runtime?: CloudFunctionRuntimes;
 } & Deployable;
 
+export type FunctionsConfig = FunctionsSingle | FunctionsSingle[];
```

Comment:
I'm not sure if we have other guidance on this or not, but why not:

```typescript
export type FunctionConfig = {}

export type FunctionsConfig = FunctionConfig | FunctionConfig[];
```

Maybe it's a little "too close" with being plural or not, but "FunctionsSingle" seems... I dunno, I wouldn't know what to expect from that.

Reply:
Lol. Naming things is hard. `Config` vs `Configs` was also a thing that was messing up my brain 🤔 .

Going ahead w/ your suggestion.